filesystem/lvm.ksy

```diff
@@ -0,0 +1,163 @@
+meta:
+  title: Logical Volume Manager
+  id: lvm
+  endian: le
+  encoding: ascii
+  application:
+    - linux
+    - grub2
+    - lvm tools
+    - libvslvm
+  license: GNU-FDL-1.3+
```

Comment:
1. That expression in SPDX would be [GFDL-1.3+](https://spdx.org/licenses/GFDL-1.3.html), not "GNU-FDL-1.3+".
2. I see that you're borrowing lots of stuff from https://github.com/libyal/libvslvm/blob/master/documentation/Logical%20Volume%20Manager%20(LVM)%20format.asciidoc — which is probably licensed on the same terms as whole libvslvm — i.e. [LGPLv3+](https://github.com/libyal/libvslvm/blob/master/COPYING). One can't just slap GFDL on that — you'll need author's permission. Have you contacted Joachim Metz? May be he'll be intersted in this ksy as well ;)

Reply:
That document is licensed under GNU Free Documentation License.

Reply:
Ah, I've missed info inside the document itself. Thanks!